paywall/unlock.2.0.js.webpack.config.js

```diff
@@ -0,0 +1,38 @@
+/* eslint no-console: 0 */
+var path = require('path')
+const webpack = require('webpack')
+const configVariables = require('./environment')
+
+module.exports = () => {
+  return {
+    cache: false,
+    mode: 'development',
```

Comment:
Even with mode set to `development` or `none`, this takes about 10 seconds to build on my machine. It may be worth simply building this with `tsc`, which should be faster and it will be easier for us to implement automatic builds on changes. For the time being this works well enough.